Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst

```diff
@@ -37,11 +37,20 @@ The syntax of argument 1 for getFileObjectFromCombinedIdentifier is
 
    [[storage uid]:]<file identifier>
 
-The storage uid is optional. If it is not specified, the default storage
-(virtual storage with uid=0) is used. In the case of a storage uid=0 the local filesystem is checked
-for the given file. If the file is found, then its local path will be used. If the file is not found,
-then the fileadmin on the public web path will be used.
-The file identifier is adapted accordingly to match the new storage's base path.
+The return value is
+
+.. code-block:: none
+
+   File|ProcessedFile|null
+
+The storage uid is optional. If it is not specified, the default storage 0 will be assumed for the beginning.
+The default storage is virtual with :php:`$uid === 0` in its class :php:`\TYPO3\CMS\Core\Resource\ResourceStorage`. In this case the local filesystem is checked for the given file. 
+The file identifier is the local path and filename relative to the TYPO3 fileadmin folder.
+Example: `/templates/stylesheets/fonts.css`, if the file `/server-path/mydomain/fileadmin/templates/stylesheets/fonts.css` exists on the file system.
```

Comment:
```suggestion
Example: `/templates/stylesheets/fonts.css`, if the file `.../fileadmin/templates/stylesheets/fonts.css` exists on the file system.
```

Reply:
This is problematic, because it does not make clear that this is an absolute path and not a relative one.

Reply:
Indeed, I see your point. Then I'd suggest `/absolute/path/to/fileadmin/...`.